README.md

```diff
@@ -73,6 +73,58 @@ cmake --build .
 cmake --build . --target package
 ```
 
+## macOS
+
+### Installing Iberty
+
+Most of SLEIGH's dependencies can be installed via the [Homebrew package manager](https://brew.sh/) on macOS. The only exception is Iberty which doesn't have a dedicated Homebrew package. Instead, we can edit the `binutils` package to include an Iberty installation.
+
+Firstly, we need to edit the Binutils installation script:
+
+```sh
+brew edit binutils
+```
+
+The command above will open the installation script with the editor specified by `EDITOR`. We need to add the `--enable-install-libiberty` flag to the `configure` invocation:
```

Comment:
Hmmm. I'm now wondering if maybe we should follow [libiberty's recommendation on vendoring](https://gcc.gnu.org/onlinedocs/libiberty/Using.html#Using) it instead of editing Homebrew.

Looking at the formulas in Homebrew, I see that `distcc` requires libiberty and has some special handling https://github.com/Homebrew/homebrew-core/blob/125fa1050f2ab030d13e0c189fe406690e5465b0/Formula/distcc.rb#L27-L43

Thoughts?